Composer/packages/client/src/components/Notifications/__tests__/useSurveyNotification.test.tsx

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React from 'react';
+
+import { renderWithRecoil } from '../../../../__tests__/testUtils';
+import { NotificationContainer } from '../NotificationContainer';
+import { useSurveyNotification } from '../useSurveyNotification';
+import { machineInfoState } from '../../../recoilModel';
+import { ClientStorage } from '../../../utils/storage';
+import { LAST_SURVEY_KEY } from '../../../constants';
+
+let savedVersion: string | undefined = '';
+const MOCK_VERSION = '2.3.4_jest';
+
+let surveyStorage;
+
+beforeAll(() => {
+  process.env.NODE_ENV = 'jest';
+  savedVersion = process.env.COMPOSER_VERSION;
+  process.env.COMPOSER_VERSION = MOCK_VERSION;
+
+  surveyStorage = new ClientStorage(window.localStorage, 'survey');
```

Comment:
You should mock local storage

Reply:
For what I'm doing here with it, there's no need to; I don't need to check that stuff on localStorage is behaving right outside of the behaviors we see in the notification via the ClientStorage class, and `window.localStorage` is safe because JSDom provides it in a way that doesn't cause interference between different test files.